src/client/js/Utils/InterpreterManager.js

```diff
@@ -96,14 +96,15 @@ define([
 
                 // Before executing the plugin - make sure the client is in SYNC.
                 // (If not plugin could be executed on the server on a commitHash that does not exist).
-                if (self._client.getBranchStatus() &&
-                    self._client.getBranchStatus() !== self._client.CONSTANTS.BRANCH_STATUS.SYNC) {
-                    callback(self.getPluginErrorResult(metadata.id, 'Not allowed to invoke plugin while local branch' +
-                        ' is AHEAD or PULLING changes from server.', startTime));
-                    return;
-                }
-
                 if (globalConfig.runOnServer === true) {
```

Comment:
Updated this to make it more legible and to address this issue in the latest commit!